### DISCUSSION TITLE
Adding default values for Decal_X and Decal_Y

### DISCUSSION THREAD

**Discussion by Jdance-Media** [SmartlyDressedGames/Unturned-3.x-Community#4160] [Ideas]
Noticed that when a user was creating a decal, their decal would be invisible in the editor if they did not specify the Decal_X and Decal_Y values. It might be wise to just have these values default to 1 if they are not specified, that way they can at least debug if the decal is setup properly.

This also brings up perhaps reviewing some of the old keys and seeing if it is possible for them to have default values. The only problem with having default values for basically everything is that one may never find out about a particular key.

**Comment by SDGNelson** [comment]
Good call! I'll add defaults for these.
